src/services/recipe.service.ts

```diff
@@ -14,8 +14,18 @@ export class RecipeService {
     limit: number
   ): Promise<RecipeDocument[]> {
     let recipes;
+    // admin search
     if (!userId) {
       recipes = await Recipe.find({}, null, { skip, limit });
+      if (name === null) {
```

Comment:
For falsy arguments use `if (!name)` check, unless you are explicitly checking for `null`